packages/labs/analyzer/src/lib/javascript/modules.ts

```diff
@@ -312,18 +322,22 @@ export const getPathForModuleSpecifier = (
   specifier: string,
   location: ts.Node,
   analyzer: AnalyzerInterface
-): AbsolutePath => {
+): AbsolutePath | undefined => {
   const resolvedPath = ts.resolveModuleName(
     specifier,
     location.getSourceFile().fileName,
     analyzer.commandLine.options,
     analyzer.fs
   ).resolvedModule?.resolvedFileName;
   if (resolvedPath === undefined) {
-    throw new DiagnosticsError(
-      location,
-      `Could not resolve specifier ${specifier} to filesystem path.`
+    analyzer.diagnostics.push(
+      createDiagnostic({
+        node: location,
+        message: `Could not resolve specifier ${specifier} to filesystem path.`,
+        category: ts.DiagnosticCategory.Error,
```

Comment:
I wonder if this is always an error... I feel like a lot of systems play fast and loose with specifiers and some non-standard ones will trigger this often. What happens with a specifier that's covered by a `declare module` statement too? Does it resolve to that declaration?